# Publish a pair of vectors of dimention 6x1 that contains the angles of every joint

float32[] tc_angles
float32[] ctr_angles
